CONTRIBUTING.md

```diff
@@ -1,32 +1,23 @@
-* [License inclusion principles](DOCS/license-inclusion-principles.md) 
+# Contributing
+The SPDX License List is maintained by the [SPDX legal team](https://spdx.dev/engage/participate/legal-team/). This repo is used for the submission and review of licenses, as well as maintaining [documentation](DOCS) related to the SPDX License List. Suggestions with a larger scope, cross-functional issues, and questions should be directed to the [SPDX-legal mailing list](https://lists.spdx.org/g/Spdx-legal). 
 
-* [Explanation of the fields contained on the list](DOCS/license-fields.md)
+## Submitting a license for consideration
+To submit a license for consideration for the SPDX License List, follow the guidance at [](request-new-license.md). Please be sure to review the relevant documentation prior to you submission, notably, the [license inclusion principles](DOCS/license-inclusion-principles.md) Each license submission requires review and incomplete or inappropriate submissions only take away the precious time of a small number of volunteers from other tasks.
 
-* [SPDX License List Matching Guidelines](https://spdx.github.io/spdx-spec/latest/license-matching-guidelines-and-templates/) provides guidelines to be used for the purposes of matching licenses and license exceptions against those included on the SPDX License List.
+## Minor updates
+Minor changes such as markup and typo fixes may be submitted directly to this repository (either as issues or pull-requests) without previous discussion. 
+If you find a broken URL for a license, you may make a pull request to add the new URL, but please note our policy to [retain old URLs](DOCS/license-fields.md), which can be helpful for matching. 
 
-* [SPDX Specification](https://spdx.dev/use/specifications/): It is helpful to be familiar with certain sections of the SPDX Specification that use or deal with the SPDX License List. In particular: sub-sections related to license information in Section 7, 8, and 10; Appendices B, D, and E.
+## Improvements to documentation
+Newcomers have a great perspective as to the effectiveness of documentation! You can make suggestions via an issue, if you want to discuss the changes. If there is something needs updating, a typo, or other minor improvements, feel free to go ahead and create a PR!
 
-# Join the mailing list and our bi-weekly calls
+## Reviewing submitted licenses
+Once you have familiarized yourself with the [License inclusion principles](DOCS/license-inclusion-principles.md), comment on new license requests as to whether you think the license should be included on the SPDX License List in any issue labeled ["new license/exception request"](https://github.com/spdx/license-list-XML/issues?q=is%3Aopen+is%3Aissue+label%3A%22new+license%2Fexception+request%22+-label%3A%22new+license%2Fexception%3A+Accepted%22)
```

Comment:
Maybe rather than just "whether you think the license should be included," instead something like "whether you think the license _meets the License Inclusion Principles_ and should be included..."

Reply:
missing period at end of sentence